workshop2/01-setup.md

```diff
@@ -0,0 +1,83 @@
+
+If you still have the repo cloned locally, you can continue with the next section. If you delete it, you can clone it again. For this open up your preferred shell:
+
+1. Create a local folder to keep your code and change to it. For example, to
+   create `C:\CodessInTheClassroom`:
+
+       mkdir C:\CodessInTheClassroom
+       cd C:\CodessInTheClassroom
+
+2. Find your repository within the [CodessInTheClassroom GitHub organisation].
+
+3. Clone your repository using the instructions in [our Git
+   cheatsheet](../git-cheatsheet.md#clone). For example:
+
+       git clone https://github.com/codessintheclassroom/happy-rover.git
+
+### Creating a branch for this workshop work
+
+Now we'll create a new branch to hold all the new code for the REST API. This will allow to submit a pull request at the end and get your code review.
+
+1. [Follow the instructions for creating a branch in the Git
+   cheatsheet](../git-cheatsheet.md#create-a-branch). You should give it a
+   descriptive name so that you know what it's for, such as
+   `yourname/shelter-backend` (on some teams developers prefix their branch
+   names with their usernames) or `shelter-backend`.
+
+### Creating the initial application code
+
+Now we'll use [.NET Core CLI] to create the initial code for our
+application:
+
+1. Using your favourite shell, go to the cloned repo location, for example, if you repository was called `happy-rover`, run `cd c:\CodessInTheClassroom\happy-rover` and run the following command to  generate a web api project `dotnet new webapi -n backend`. This will create a new C# project with some example code for a weather forecast api, that we will override to create our animal shelter API.
+2. Run `code -r c:\CodessInTheClassroom\happy-rover\backend` to open the new created project in Visual Studio Code.
+If you're interested in what other options you can pass when creating a project using `.NET Core CLI` , you can [read more here](https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-new).
+3. Let's test now the generated web api, hit `Ctrl+F5` in Visual Studio Code, and it will compile and run the project. If there are no errors, in a browser go to the following URL: [https://localhost:5001/WeatherForecast](https://localhost:5001/WeatherForecast), and the server will return a json document with the weather forecat for the next week.
+
+### Commit the code changes and push to GitHub
+
+So far, we've generated a very basic web api, that we will use to create our animal shelter backend. The `dotnet new webapi` command has created a few files in the repository folder that will be the base for our application.
+
+We'll commit and push this code to our development branch to have a start point. To do that:
+1. [Read the guidance on staging changes and committing in the Git cheatsheet](../git-cheatsheet.md#commit).
+
+   For example, you could open a new terminal using `Terminal → New Terminal` and run:

```

Comment:
(at this point they have already opened a PoSh or cmd prompt, so this might be a bit confusing)